components/section/AboutSection/About.jsx

```diff
@@ -0,0 +1,55 @@
+import React, { useState } from 'react';
+import { OneTab } from '../../functions/OneTab';
+import PageLink from '../../ui/Link';
+import Svg from '../../ui/Svg';
+import Wrapper from '../../wrappers/Wrapper';
+const tabTexts = [
+  { id: 0, text: 'Tab 1', content:  <div>About us Blockchain will do for cooperation what the internet did for communication - We believe that blockchain technology has the potential to shift financial models as we know them, creating revenue streams that are more sustainable and fair for users. We envision a future in which financial infrastructures are rather decentralised and horizontal, in which any member can have a say. Through its immutability, blockchain technology provides trustworthy infrastructures that allow cooperation on a large scale.  <PageLink text="Learn more" style="gap-2 " textStyle="underline" arrow=<Svg />/> <br /> Work Meaningful - With our work, we aim to impact society positively. The projects we get involved with the aim to solve significant problems in our society, sustainably and in the long run, rather than looking for a quick profit through speculation or encouraging harmful behavior.</div>},
+  { id: 1, text: 'Tab 2', content:  <div>What we do Blockchain will do for cooperation what the internet did for communication - We believe that blockchain technology has the potential to shift financial models as we know them, creating revenue streams that are more sustainable and fair for users. We envision a future in which financial infrastructures are rather decentralised and horizontal, in which any member can have a say. Through its immutability, blockchain technology provides trustworthy infrastructures that allow cooperation on a large scale.  <PageLink text="Learn more" style="gap-2" textStyle="underline" arrow=<Svg />/> <br /> Work Meaningful - With our work, we aim to impact society positively. The projects we get involved with the aim to solve significant problems in our society, sustainably and in the long run, rather than looking for a quick profit through speculation or encouraging harmful behavior.</div>},
+  { id: 2, text: 'Tab 3', content:  <div>Hiring Blockchain will do for cooperation what the internet did for communication - We believe that blockchain technology has the potential to shift financial models as we know them, creating revenue streams that are more sustainable and fair for users. We envision a future in which financial infrastructures are rather decentralised and horizontal, in which any member can have a say. Through its immutability, blockchain technology provides trustworthy infrastructures that allow cooperation on a large scale.  <PageLink text="Learn more" style="gap-2" textStyle="underline" arrow=<Svg />/> </div>},
+  
+];
+
+
+export default function About (){
```

Comment:
You can try formating your code with editor formaters in order to format your code